### DISCUSSION TITLE
Want to remove the Header and the Authorize Button

### DISCUSSION THREAD

**Discussion by Jonak-Adipta-Kalita** [jellydn/next-swagger-doc#209] [General]
How can I remove the Part where it says `<YOUR_API_NAME> VERSION` and also the Authorize button?
![Screenshot (353)](https://user-images.githubusercontent.com/70377522/159617817-19f7c5a1-5869-4a71-b2ee-552673d3b2d5.png)

**Comment by Jonak-Adipta-Kalita** [comment]
Please I need help :pray:

**Reply by Jonak-Adipta-Kalita** [reply]
Please Help!!!! I really need it :pray:

**Reply by Jonak-Adipta-Kalita** [reply]
Nevermind, got a solution

**Reply by jellydn** [reply]
Great. How did you resolve this one? Thanks.

**Reply by Jonak-Adipta-Kalita** [reply]
I just targeted the styles and added `display: none;`
